ui/src/utils/TemporaryFile.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as crypto from "crypto";
+import * as fse from 'fs-extra';
+import * as os from 'os';
+import * as path from 'path';
+
+// tslint:disable-next-line:no-stateless-class export-name

```

Comment:
Rather than disable the rule here, you could just make it a static class or a namespace